src/App.jsx

```diff
@@ -0,0 +1,35 @@
+import React, {useState, useEffect, useMemo} from 'react';
+import ReactDOM from 'react-dom';
+import {HashRouter, Link} from 'react-router-dom';
+import Editor from './components/Editor.jsx';
+
+const App = () => {
+
+    //Uses some syntax for React-Router functionality, but I didn't get to that stretch goal
+    return (
+        <div className={'main-container'}>
+                <div className={'navigation'}>
+                    <div className={'nav-wrapper'}>
+                        <div className={'main-title'}>Kytra's Live CSS Editor</div>
+                        <div className={'nav-links-wrapper'}>
+                            <div className={'nav-links'}>
+                                <Link to={'/'}>Editor</Link>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div className={'content-container'}>
+                    <div className={'editor-wrapper'}>
```

Comment:
You do not need `{}` around props that don't need computation (such as string literals).
`<div className="content-container" />` is better.